components/ui/eventsCategorized/index.tsx

```
@@ -140,19 +136,26 @@ function EventsCategorized() {
   if (error) return <NoEventsFound title={notFoundText} />;
 
-  let priorityCount = 0;
-
   // Render
   return (
     <>
+      {/* JSON-LD structured data for SEO */}
       <Script
         id={`${place || "catalunya"}-${byDate || "all"}-script`}
         type="application/ld+json"
         dangerouslySetInnerHTML={{ __html: JSON.stringify(jsonEvents) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)